### DISCUSSION TITLE
Add the ability to run now a Schedule in the new client area

### DISCUSSION THREAD

**Discussion by Czekoloczek** [pelican-dev/panel#793] [Ideas]
Add the ability to run now a Schedule in the new client area

Run now :D
![obraz](https://github.com/user-attachments/assets/03a4c99c-ecd3-46da-b4ad-8a44d451ec82)

No run now :(
![obraz](https://github.com/user-attachments/assets/d09376f8-1141-4553-a240-d9019ae4a747)

**Comment by rmartinoscar** [comment]
Please follow #794
